roles/pulp_api/tasks/main.yml

```diff
@@ -14,6 +14,17 @@
         # We can also try using seport's ignore_selinux_state variable
         - ansible_facts.selinux.status == "enabled"
 
+    - name: Install pulpcore-api socket file (when it applies)
+      template:
+        src: pulpcore-api.socket.j2
+        dest: /lib/systemd/system/pulpcore-api.socket
```

Comment:
In the Puppet module we write to `/etc/systemd` since `/lib` is owned by packages.

Reply:
Not touching that in this PR has this has been setup like that (before this change)

Reply:
Fair enough. It's more important to be consistent.